contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -31,8 +31,11 @@ contract RaidenMicroTransferChannels {
 
     // 28 (deposit) + 4 (block no settlement)
     struct Channel {
-        uint192 deposit; // mAX 2^192 == 2^6 * 2^18
-        uint32 open_block_number; // UNIQUE for participants to prevent replay of messages in later channels
+        // mAX 2^192 == 2^6 * 2^18
```

Comment:
don't understand the comment, can it be clarified?

Reply:
Yes, didn't notice that the comment is very weird. The idea was to use the smallest `uint` size possible for `deposit`. 
10^8 * 10^18 max tokens (supply) -> 24 bytes -> uint192

Changing comment to:
```
// uint192 is the maximum uint size needed for deposit based on a 10^8 * 10^18 token totalSupply
```